2021-11-21 21:17:00,313 INFO    MainThread:13516 [wandb_setup.py:_flush():71] setting env: {}
2021-11-21 21:17:00,313 INFO    MainThread:13516 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-21 21:17:00,314 INFO    MainThread:13516 [wandb_init.py:_log_setup():357] Logging user logs to c:\Users\ruben\Downloads\VDoomLearning\src\wandb\run-20211121_211700-3pru9hn9\logs\debug.log
2021-11-21 21:17:00,314 INFO    MainThread:13516 [wandb_init.py:_log_setup():358] Logging internal logs to c:\Users\ruben\Downloads\VDoomLearning\src\wandb\run-20211121_211700-3pru9hn9\logs\debug-internal.log
2021-11-21 21:17:00,314 INFO    MainThread:13516 [wandb_init.py:_jupyter_setup():307] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000246FFCABD68>
2021-11-21 21:17:00,315 INFO    MainThread:13516 [wandb_init.py:init():390] calling init triggers
2021-11-21 21:17:00,315 INFO    MainThread:13516 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-11-21 21:17:00,315 INFO    MainThread:13516 [wandb_init.py:init():435] starting backend
2021-11-21 21:17:00,315 INFO    MainThread:13516 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2021-11-21 21:17:00,318 INFO    MainThread:13516 [backend.py:ensure_launched():199] starting backend process...
2021-11-21 21:17:00,413 INFO    MainThread:13516 [backend.py:ensure_launched():205] started backend process with pid: 15364
2021-11-21 21:17:00,413 INFO    MainThread:13516 [wandb_init.py:init():444] backend started and connected
2021-11-21 21:17:00,427 INFO    MainThread:13516 [wandb_run.py:_label_probe_notebook():866] probe notebook
2021-11-21 21:17:00,427 INFO    MainThread:13516 [wandb_run.py:_label_probe_notebook():876] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-11-21 21:17:00,428 INFO    MainThread:13516 [wandb_init.py:init():503] updated telemetry
2021-11-21 21:17:00,472 INFO    MainThread:13516 [wandb_init.py:init():533] communicating current version
2021-11-21 21:17:03,333 INFO    MainThread:13516 [wandb_init.py:init():538] got version response upgrade_message: "wandb version 0.12.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-11-21 21:17:03,334 INFO    MainThread:13516 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-21 21:17:04,121 INFO    MainThread:13516 [wandb_init.py:init():576] starting run threads in backend
2021-11-21 21:17:04,382 INFO    MainThread:13516 [wandb_run.py:_console_start():1693] atexit reg
2021-11-21 21:17:04,383 INFO    MainThread:13516 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.WRAP
2021-11-21 21:17:04,384 INFO    MainThread:13516 [wandb_run.py:_redirect():1604] Wrapping output streams.
2021-11-21 21:17:04,386 INFO    MainThread:13516 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-21 21:17:04,386 INFO    MainThread:13516 [wandb_init.py:init():603] run started, returning control to user process
2021-11-21 21:17:04,386 INFO    MainThread:13516 [wandb_run.py:_config_callback():911] config_cb None None {}
2021-11-21 22:34:01,386 INFO    MainThread:13516 [wandb_run.py:finish():1347] finishing run teccr/VDoomLearning/3pru9hn9
2021-11-21 22:34:01,388 INFO    MainThread:13516 [jupyter.py:save_history():429] not saving jupyter history
2021-11-21 22:34:01,389 INFO    MainThread:13516 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-21 22:34:01,389 INFO    MainThread:13516 [wandb_init.py:_jupyter_teardown():289] cleaning up jupyter logic
2021-11-21 22:34:01,389 INFO    MainThread:13516 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 0
2021-11-21 22:34:01,390 INFO    MainThread:13516 [wandb_run.py:_restore():1635] restore
2021-11-21 22:34:02,086 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1112
  total_bytes: 1112
}

2021-11-21 22:34:02,208 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1112
  total_bytes: 1112
}

2021-11-21 22:34:02,654 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1112
  total_bytes: 1112
}

2021-11-21 22:34:03,107 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1112
  total_bytes: 8219
}

2021-11-21 22:34:03,211 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1112
  total_bytes: 8500
}

2021-11-21 22:34:03,320 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1112
  total_bytes: 8500
}

2021-11-21 22:34:03,428 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1112
  total_bytes: 8500
}

2021-11-21 22:34:03,551 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1112
  total_bytes: 8500
}

2021-11-21 22:34:03,658 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1112
  total_bytes: 8500
}

2021-11-21 22:34:03,765 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1112
  total_bytes: 8500
}

2021-11-21 22:34:03,875 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1112
  total_bytes: 8500
}

2021-11-21 22:34:03,986 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8500
  total_bytes: 8500
}

2021-11-21 22:34:04,101 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8500
  total_bytes: 8500
}

2021-11-21 22:34:04,209 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8500
  total_bytes: 8500
}

2021-11-21 22:34:04,325 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8500
  total_bytes: 8500
}

2021-11-21 22:34:04,434 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8500
  total_bytes: 8500
}

2021-11-21 22:34:04,545 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8500
  total_bytes: 8500
}

2021-11-21 22:34:04,892 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8500
  total_bytes: 8500
}

2021-11-21 22:34:05,816 INFO    MainThread:13516 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8500
  total_bytes: 8500
}
local_info {
}

2021-11-21 22:34:07,110 INFO    MainThread:13516 [wandb_run.py:_append_history():2011] rendering history
2021-11-21 22:34:07,110 INFO    MainThread:13516 [wandb_run.py:_append_summary():1966] rendering summary
2021-11-21 22:34:07,110 INFO    MainThread:13516 [wandb_run.py:_append_files():2061] logging synced files
